### DISCUSSION TITLE
Xray to Opentelemetry

### DISCUSSION THREAD

**Discussion by kaihendry** [open-telemetry/community#772] [Community Discussions]
IIUC AWS SQS does emit traces to Xray.

Can that trace be exported from Xray into Opentelemetry? I am unsure if this use case is possible.

**Comment by SergeyKanzhelev** [comment]
CC: @alolita @rakyll

**Reply by anuraaga** [reply]
Hi @kaihendry - X-Ray is able to infer SQS in service maps based on client spans, but there currently isn't any data emitted from SQS into X-Ray. This may change in the future.

Unrelated to SQS, but if you did have a need to fetch data from X-Ray to inject into another backend via OpenTelemetry, you can use the fetching APIs such as [BatchGetTraces](https://docs.aws.amazon.com/xray/latest/api/API_BatchGetTraces.html). For example, API gateway spans could be retrieved using this API if you needed them.

**Reply by kaihendry** [reply]
Thank you, that was the answer I was seeking!
